data/interfaces/default/settings.html

```diff
@@ -813,6 +813,16 @@ <h3>Server Monitoring</h3>
                                         <input type="hidden" class="form-control" id="pms_update_distro" name="pms_update_distro">
                                     </div>
                                 </div>
+                                <div class="row">
```

Comment:
This `<div>` needs to move one level up to a new `<div class="form-group">` otherwise the spacing is inconsistent with everything else. And the `<label>` needs to move above the `<div class="row">`.

![image](https://user-images.githubusercontent.com/9099342/56087174-60573780-5e1a-11e9-838b-be7e413d0b7b.png)


```suggestion
                            </div>
                            <div class="form-group">
                                <label for="pms_update_check_interval">Update Check Interval</label>
                                <div class="row">
                                    <div class="col-md-2">
                                        <input type="text" class="form-control" data-parsley-type="integer" id="pms_update_check_interval" name="pms_update_check_interval" value="${config['pms_update_check_interval']}" size="3" data-parsley-min="1" data-parsley-trigger="change" data-parsley-errors-container="#pms_update_check_interval_error" required>
                                    </div>
                                    <div id="pms_update_check_interval_error" class="alert alert-danger settings-alert" role="alert"></div>
                                </div>
                                <p class="help-block">
                                    The interval (in hours) Tautulli will check for a new Plex Media Server update. Minimum 1, default 24.
                                </p>
```